Localize/localize-pipeline.yml

```diff
@@ -23,6 +37,20 @@ stages:
       env:
           SYSTEM_ACCESSTOKEN: $(System.AccessToken)
 
+    - powershell: |
+        $body = '{"text": "Created tasks localization update PR. Someone please approve/merge it. :please-puss-in-boots:"}'
+        Invoke-RestMethod -Uri $(slackUri) -Method Post -Body $body -ContentType 'application/json'
+      displayName: 'Send Slack notification'
+      condition: and(succeeded(), eq(variables['WEEK'], '3'), eq(variables['build.reason'], 'Schedule'))
+
+    - powershell: |
+        $buildUrl = "$(System.TeamFoundationCollectionUri)$(System.TeamProject)/_build/results?buildId=$(Build.BuildId)&_a=summary"
+        $body = '{"text": "Something went wrong while creating tasks localization update PR. Build: ' + $buildUrl + '"}'
+        Invoke-RestMethod -Uri $(slackUri) -Method Post -Body $body -ContentType 'application/json'
+      displayName: 'Send Slack notification'

```

Comment:
Nit: could you please update display name to make it more precise? (e.g. 'Send Slack notification about error')

Reply:
Sure! Thank you for the point. I'll apply this to all repos.